### PR TITLE
ENG-601: Update packages on push to polybase-ts repo

### PR DESCRIPTION
**Pull request by soru23**

### CODE REVIEW

File: .github/workflows/update-beta-packages.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        with:
+          node-version: 18
+
+      - name: Install yarn
+        run: sudo npm install -g yarn
+      
+      - name: Install cargo
+        uses: baptiste0928/cargo-install@v2
+        with:
+          crate: cargo
```
Comment (calummoore) [inline .github/workflows/update-beta-packages.yml:26]: Cargo is not needed for this repo

File: .github/workflows/update-beta-packages.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Add npm package
+        run: yarn add @polybase/polylang@beta
```
Comment (calummoore) [inline .github/workflows/update-beta-packages.yml:29]: auth does not use polylang, have a look at the package.json, for this one its:

  "@polybase/client"
  "@polybase/eth"
  "@polybase/react"
  "@polybase/util"

**Comment by calummoore** [comment]
Can you add a trigger to run on push, so we can test this out. And then I will test the triggering of dispatch via publishing the other one.

**Comment by calummoore** [comment]
@soru23 

Please resolve comments when you fix them 😄.

> Can you add a trigger to run on push, so we can test this out. And then I will test the triggering of dispatch via publishing the other one.

By this I meant adding the current branch as a push trigger, test that it works, and then revert the change. That way we know this action (once called) would work as expected (which I don't think it does). I.e. it's creating a new branch, but you're not committing any of the changes to the new branch. Take a look at the other Github workflows already in there, as we already create automated PRs.

Also, to avoid conflicts, we should create a branch which includes the version number. This can actually be sent as a parameter using repository_dispatch.

**Comment by calummoore** [comment]
Well it turns out it did apply the updates to the PR, nice - https://github.com/polybase/auth/pull/17

So we just need to include the version number in the title and branch name :)

**Comment by soru23** [comment]
ok cool let me see how we can include that

**Review by calummoore — APPROVED** [review]

**Pull request merged**